sites/mosul/configs/openmrs/initializer_config/drugs/drugs.csv

```diff
@@ -1,7 +1,758 @@
+"fc71cb58-97e1-4fbf-83e4-820ddcebac75","MORPHINE sulfate, 10mg, prolonged-release caps.","28992602-98a8-492d-9c3a-816da76bbbc1","08cdfd23-4dca-4f2a-967d-0fd617075e6e","10mg"
+"709d7e4e-d48a-4cdd-bee3-256125efefe2","MORPHINE sulfate, 10mg/5ml, oral sol., 100ml, bot.","28992602-98a8-492d-9c3a-816da76bbbc1","32e02e8e-a4b6-472b-9c9b-2afbc3a6cb78","10mg/5ml, 100ml"
+"823d0dd4-64cf-4128-a4af-534cf28086d4","MORPHINE sulfate, 10mg, immediate-release, tab.","28992602-98a8-492d-9c3a-816da76bbbc1","81d0f25d-4721-40cb-94ff-99601a260b28","10mg"
+"5bcab381-030e-4ae9-a8e8-b8af88c4b878","MORPHINE sulfate, 10mg, prolonged-release tab.","28992602-98a8-492d-9c3a-816da76bbbc1","b9f5995a-fe7e-4901-8f36-a0277cd0bb14","10mg"
+"0f45f52a-2b83-4515-a056-6a23361ce810","MORPHINE sulfate, 30mg, prolonged-release caps.","28992602-98a8-492d-9c3a-816da76bbbc1","08cdfd23-4dca-4f2a-967d-0fd617075e6e","30mg"
+"d33a928d-a688-45fe-ab19-d0721682dae2","MORPHINE sulfate, 30mg, prolonged-release, tab.","28992602-98a8-492d-9c3a-816da76bbbc1","b9f5995a-fe7e-4901-8f36-a0277cd0bb14","30mg"
+"e710b92c-d322-4621-a5db-65d3e81e6acf","MOXIFLOXACIN hydrochloride, eq.100mg base, disp.tab. blister","3fb99832-8748-4b9e-a7cc-f6b46cb11ced","06fa5b83-7472-45c4-94dc-8b06075690ba","eq.100mg base"
+"bb416b0c-9990-42d3-99a8-a2089202cca9","MOXIFLOXACIN hydrochloride, eq 400mg base, tab. blister","3fb99832-8748-4b9e-a7cc-f6b46cb11ced","c1f4a088-32a8-4b34-8162-944bb6ed026f","eq 400mg base"
+"2818d63c-689c-4c61-a678-5253da1e5122","MULTIVITAMINS, tab.","e66f9a49-ce1e-4477-99d6-513db8316110","c1f4a088-32a8-4b34-8162-944bb6ed026f",
+"bafa2684-a182-4e43-b8dd-4f7fbac3d6a0","NEVIRAPINE (NVP), 50mg/5ml, oral susp., 100ml, bot.","2f529a3c-5ca6-43aa-ad4c-9c61a0696f08","fe152bf2-f35e-41fa-bc78-bbb94b60d6d2","50mg/5ml, 100ml"
+"1acc05ae-c343-45e5-a696-9f1845d34d1a","NEVIRAPINE (NVP), 50mg/5ml, oral susp., 240ml, bot.","2f529a3c-5ca6-43aa-ad4c-9c61a0696f08","fe152bf2-f35e-41fa-bc78-bbb94b60d6d2","50mg/5ml, 240ml"
+"37ce5f8b-89b7-4052-8075-c807cd5c68d2","NEVIRAPINE (NVP), 50mg, disp. tab.","2f529a3c-5ca6-43aa-ad4c-9c61a0696f08","06fa5b83-7472-45c4-94dc-8b06075690ba","50mg"
+"9a1bacbf-5fb4-43f8-938b-92681641e976","NICOTINAMIDE (vitamin PP), 100mg, tab.","608fd7d5-6d3a-434b-a42b-a25f062bf7c3","c1f4a088-32a8-4b34-8162-944bb6ed026f","100mg"
+"8101ead5-0c6d-40e5-a7f3-eb69d69b2212","NIFEDIPINE, 10mg, immediate-release tab.","35a9ad57-bdd1-4f43-b355-d22bdfbd3250","81d0f25d-4721-40cb-94ff-99601a260b28","10mg"
+"0269a9c2-1000-47d2-ba2b-a5f5553e1a82","NIFURTIMOX, 120mg, tab.","892604bb-0989-421d-8df0-76866079bfd9","c1f4a088-32a8-4b34-8162-944bb6ed026f","120mg"
+"b1ebb531-ead3-4744-af95-6609ea5d0bca","NITROFURANTOIN, 100mg, caps.","e46d140d-ba91-4234-b0cc-8f33ef714ad1","c2c7e944-e9d6-4b2d-bf1e-b542dbdbb634","100mg"
+"d6658c05-d03b-407c-8f33-f67eb2edd9ad","NYSTATIN, 100,000 IU/ml, oral susp. + graduated dropper","1806171f-7506-4418-b3f2-25290ff86a14","fe152bf2-f35e-41fa-bc78-bbb94b60d6d2","100,000 IU/ml"

```

Comment:
This is erroring but it's a false positive as `1806171f-7506-4418-b3f2-25290ff86a14` exists in OCL [here](https://app.openconceptlab.org/#/orgs/MSF/sources/MSF/?q=1806171f-7506-4418-b3f2-25290ff86a14&isTable=true&isList=false&page=1)
<img width="1460" alt="Screenshot 2025-05-29 at 15 37 07" src="https://github.com/user-attachments/assets/c91f6b21-8f11-4c12-8e3a-35cdfc440d06" />